modape/scripts/modis_download.py

```diff
@@ -69,6 +73,8 @@ def cli(products: List[str],
         strict_dates (bool): Strict date handling.
         download (bool): Download data.
```

Comment:
Actually `def cli()` returns `None` for `download == False`. Also, I think the "Returns" description needs to be updated: a List of *actually downloaded* MODIS HDF file names is returned

Reply:
Good point.

I think we need to be more explicit in case there's no download. Either we _specifically_ return `None` or an empty list. I'm leaning for option no.2

Reply:
+1 for option 2